pwa-devdocs/src/technologies/upgrading-versions/index.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Upgrading versions
+---
+
+Upgrading to the latest PWA Studio version gives your project access to the new features, bugfixes, and improvements associated with that release.
+This topic provides general guidance and best practices for upgrading your storefront project to use the newest PWA Studio release.
+
+## Prerequisites
+
+The contents of this topic applies to storefront projects setup according to the [Setup your project][] tutorial.
+These projects use the `venia-concept` package as a template to set up the directory structure and project dependencies.
+This is the recommended setup for new projects because it makes version upgrades easier.
+
+[setup your project]: <{% link /tutorials/pwa-studio-fundamentals/project-setup/index.md %}>
+
+## Update dependencies
+
+The main updates you need to make in your storefront project are the dependency versions in your `package.json` file.
+PWA Studio packages are independently versioned, which means their versions may not be the same as the main PWA Studio release version.
+
+If your project uses `yarn`, you can call `yarn add` on each of the `@magento` packages.
+This automatically updates the `package.json` file in your project and install the latest versions.
+
+Sample command:
+
+```sh
+yarn add @magento/eslint-config @magento/pagebuilder @magento/peregrine @magento/pwa-buildpack @magento/upward-js @magento/venia-ui
+```
+
+If your project uses `npm`, you can still manually edit the `package.json` file with the latest published version.
+Compare the contents of your project's `package.json` file with the one in `venia-concept` to see the latest published versions.
+For example, the `package.json` file for the **v10.0.0** release is at:
+
+<https://github.com/magento/pwa-studio/blob/v10.0.0/packages/venia-concept/package.json>
```

Comment:
I feel like this comment should have more visibility as this is how we're informing them to find the version to upgrade.

Could we maybe include a short example of finding the new version, modifying it in your project and running an update?